client/app/src/accounts/account.controller.js

```diff
@@ -1275,14 +1276,31 @@
           .cancel(gettextCatalog.getString('Cancel'))
         $mdDialog.show(confirm).then(() => {
           networkService.removeNetwork(network)
-          self.listNetworks = networkService.getNetworks()
-          toastService.success(gettext('Network removed succesfully!'), 3000)
+          if (isActive) {
+            dialogService.openLoadingDialog(self.currentTheme,
+                                            gettext('Network removed succesfully'),
+                                            gettext('Network removed succesfully - will now switch the network'))
```

Comment:
"successfully" is spelt incorrectly. Also, i'm not sure how much work it would be, but maybe changing it so it's not a loading dialog but a dialog that tells them the wallet needs to reload, with a reload button. then it just reloads on close 🤷‍♂️

Reply:
Well problem is, that this is a generic loading dialog which I did either reuse here or reuse in another place :P I find for this situation it's enough like that (because almost nobody ever deletes a network) - but of course in the end it's your decision :)

Reply:
A simple solution would be adding a new parameter to `openLoadingDialog` that permits customizing its behaviour or content.

2 examples:
 - Using `templateUrl`, to add there the reload button.
 - Passing a `options` parameter that enables HTML instead of pure text (and change the template too).

Reply:
I guess I just fail to see the advantage of showing a button to the user. He cannot do anything anyway - he HAS to reload because the network was deleted and the dialog is modal.

Reply:
To me the UX didn't feel seamless. I'd rather a button I have to press to reload, because then I can press it in my own time. Loading screens I feel are used for processing in the background or waiting for something to happen, and not to display a message to the user, because I might be a slow reader who didn't manage to read the whole message before it disappeared.

Reply:
Okay, that's a valid point.
I will adjust it accordingly.

Reply:
Done - what do you think?

![image](https://user-images.githubusercontent.com/1086065/36629296-481efd80-1954-11e8-9a59-55810e270d7f.png)

Reply:
Looks good! I'll have a cheeky test Monday